### ISSUE TITLE
ANSI-TESTS: The symbol COMMON-LISP::LOGICAL-PATHNAME has no function bound to it

### ISSUE THREAD

**Issue by kpoeck** [clasp-developers/clasp#408]
Although logical-pathnames in principle seem to be implemented, LOGICAL-PATHNAME seems to be forgotten.
GOOD:
````
COMMON-LISP-USER>  (logical-pathname-translations "sys")
((#P"SYS:**;*.*" #P"/Users/karstenpoeck/lisp/compiler/clasp2/src/lisp/**/*.*"))
````
BAD:
````
(setf (logical-pathname-translations "foo") (list (list #P"SYS:**;*.*" #P"/Users/karstenpoeck/lisp/compiler/clasp2/src/lisp/**/*.*")))
((#P"SYS:**;*.*" #P"/Users/karstenpoeck/lisp/compiler/clasp2/src/lisp/**/*.*"))
(logical-pathname "foo:bar;nada.lisp")
->
Condition of type: SIMPLE-PROGRAM-ERROR
The symbol COMMON-LISP::LOGICAL-PATHNAME has no function bound to it
````
In SBCL:
````
* (logical-pathname "foo:bar;nada.lisp")
#P"FOO:BAR;NADA.LISP"
````

**Comment by kpoeck** [comment]
I think this the function exists happily, but is just not correctly exposed to the scraper, since cl_defun and an underscore is missing. Will build and test.

**Comment by kpoeck** [comment]
Fixed with pr #447, waiting for merge

**Comment by kpoeck** [comment]
Retest fine, closing
````
CL-USER> (setf (logical-pathname-translations "foo")
      (list (list #P"SYS:**;*.*"#P"/Users/karstenpoeck/lisp/compiler/clasp2/src/lisp/**/*.*")))

((#P"SYS:**;*.*" #P"/Users/karstenpoeck/lisp/compiler/clasp2/src/lisp/**/*.*"))
CL-USER> (logical-pathname "foo:bar;nada.lisp")
#P"FOO:BAR;NADA.LISP"
CL-USER> 
````
